pkgs/development/python-modules/av/default.nix

```diff
@@ -17,16 +17,16 @@
 
 buildPythonPackage rec {
   pname = "av";
-  version = "12.3.0";
+  version = "12.3.0-unstable-2024-08-19";
```

Comment:
I assume a release is imminent and we wait for that?

Otherwise I say we do this in a separate PR. Unpinning is a secondary goal, we should get everything that isn't pinned upgraded first.

Reply:
Yep, per the PR and commit message – it’s scheduled for the start of September, before the next staging cycle will start, hence this being a draft for now. The reason to do this at the same time is to deal with the vast majority of FFmpeg 7 problems at once, Python system included, rather than stretching it out over multiple cycles.